ibc/module.go

```diff
@@ -69,91 +65,36 @@ func (m *ibcModule) Start() error {
 		m.logger.Info().Msg("🚫 IBC module disabled 🚫")
 		return nil
 	}
-	m.logger.Info().Msg("🪐 starting IBC module 🪐")
-	// TODO: start the host logic
+	if m.host != nil {
+		if err := m.host.StartBackgroundTasks(m.ctx); err != nil {
+			return err
+		}
+	}
 	return nil
 }
 
 func (m *ibcModule) Stop() error {
+	m.logger.Info().Msg("🛑 Stopping IBC Module 🛑")
```

Comment:
🛑 💯

Reply:
We should do this everywhere

Reply:
Worth including in [the docs](https://github.com/pokt-network/pocket/blob/main/shared/modules/doc/README.md#stop-the-module)?